src/mpyl/steps/deploy/dagster.py

```diff
@@ -1,14 +1,14 @@
 """
 Step to deploy a dagster user code repository to k8s
 """
-
```

Comment:
You need to setup the black formatter, otherwise we'll keep having these kind of changes between you and me/Pedro 😬